.gitignore

```diff
@@ -1,5 +1,6 @@
 .phpunit.result.cache
 /vendor
+/tests/fixtures
```

Comment:
Did we mean to ignore the fixtures from PHP-VCR?

Reply:
It has already been put so that it is not ignored